java-client/session-examples/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
 
     Classpaths.inheritGrpcPlatform(project, 'implementation')
     implementation 'io.grpc:grpc-core'
+    implementation 'io.grpc:grpc-netty'
```

Comment:
It seems odd that this wasn't required before now, in that it might be a hole in the java-client-session api to not convey this, or this might be a more in-depth example than normally goes in this project?

Reply:
Sorry, I need to change this to `runtimeOnly`.

Reply:
During your BOM refactoring, the session-examples broke - I haven't dug too deep, but this is the quick fix to get it back working.